### PR TITLE
ci(gh): add 'num_services' and 'instances_per_service' inputs for GH action

### PR DESCRIPTION
**Pull request by lobkovilya**

### CODE REVIEW

File: .github/workflows/perf-test-eks.yaml [hunk 1]
```diff
@@ -6,6 +6,16 @@ on:
...
+      num_services:
+        description: "Number of services to run during testing"
+        required: false
+        type: number
+        default: 5
+      instances_per_service:
+        description: "Number of instances per service to run during testing"
+        required: false
+        type: number
+        default: 1
```
Comment (michaelbeaumont) [inline .github/workflows/perf-test-eks.yaml:18]: why have the defaults here? doesn't this make it more difficult to run locally?

Comment (lobkovilya) [reply .github/workflows/perf-test-eks.yaml:18]: I'll remove this after merging #52

File: .github/workflows/perf-test-eks.yaml [hunk 2]
```diff
@@ -35,7 +43,12 @@ jobs:
       - name: Configure kubectl
         run: aws eks --region us-west-1 update-kubeconfig --name mesh-perf
       - name: Run tests
-        run: MESH_VERSION=${{ inputs.mesh_version }} STABILIZATION_SLEEP=30s make run
+        env:
+          MESH_VERSION: ${{ inputs.mesh_version }}
```
Comment (michaelbeaumont) [inline .github/workflows/perf-test-eks.yaml:47]: maybe this should also be prefixed?

**Review by michaelbeaumont — APPROVED** [review]

File: .github/workflows/perf-test-eks.yaml (hunk 2 above)
Comment (lobkovilya) [reply .github/workflows/perf-test-eks.yaml:47]: yeah, why not, let me update the PR

**Pull request merged**